### PR TITLE
Fix/collaboratorreports

### PR DESCRIPTION
**Pull request by emelalkim**

### CODE REVIEW

**Review by OzgeYurtsever — APPROVED** [review]

File: plugins/Reporting.js [hunk 1]
```diff
@@ -1488,15 +1511,20 @@ async function reporting(fastify) {
                   if (exportCalcs[valNum].field === 'recist') recistRequired = true;
                   else longitudinalRequired = true;
                 }
-                const recistReport = recistRequired ? fastify.getRecist(aimsRes.rows) : undefined;
+                // here
```
[the remark this answers is not in the document]
Comment (emelalkim) [reply plugins/Reporting.js]: done

File: plugins/EpadDB.js [hunk 2]
```diff
@@ -8072,7 +8094,8 @@ async function epaddb(fastify, options, done) {
               subject.id,
               result.rows,
               epadAuth,
-              transaction
+              transaction,
+              false // it is admin
```
[the remark this answers is not in the document]
Comment (emelalkim) [reply plugins/EpadDB.js:8101]: already filtering. except for best response which gets the first reader's report for now

File: plugins/Reporting.js [hunk 3]
```diff
@@ -489,9 +489,19 @@ async function reporting(fastify) {
...
       const targetTypes = ['target', 'target lesion', 'resolved lesion'];
       const users = {};
+
       // first pass fill in the lesion names and study dates (x and y axis of the table)
       for (let i = 0; i < lesions.length; i += 1) {
-        const username = lesions[i].username.value.toLowerCase();
+        // check if the user is a collaborator
+        // if so only get his/her username, ignore the rest
+        const username = lesions[i].username.value;
+        if (collab && username !== request.epadAuth.username) {
```
[the remark this answers is not in the document]
Comment (emelalkim) [reply plugins/Reporting.js]: passed epadauth to the methods

**Review by emelalkim — COMMENTED** [review]
fixed all issues that were pointed out

**Pull request merged**